conf/messages.en

```diff
@@ -231,8 +231,8 @@ view.export-questions.whichPriorityGoods.title=Which type of priority goods are
 view.export-questions.whichPriorityGoods.heading=Which type of priority goods are you moving?
 view.export-questions.freightType.title=Which type of transport are you using?
 view.export-questions.freightType.heading=Which type of transport are you using?
-view.export-questions.vessel-details.title=Tell us about your transport
-view.export-questions.vessel-details.heading=Tell us about your transport
+view.export-questions.vessel-details.title=More details about your transport
```

Comment:
Change is fine though I'd noticed on the screenshot for the story the content for the vessel name field has also been updated. This wasn't explicitly reflected in the story but I don't see another one for it and doesn't seem worth raising a full story for just this - are you happy to make the change for this? 

I'm happy to add to story if so 

<img width="1041" alt="image" src="https://user-images.githubusercontent.com/55190161/101478854-3afd3280-3949-11eb-9c8f-3b735f95afa7.png">

Reply:
https://docs.google.com/spreadsheets/d/16ix1ylUxW9mymYW5QWn-NhcnSgLH05pMkIw6bcTEcBQ/edit?pli=1#gid=1645738132 

Here's a link to the spreadsheet as well in case you don't have it (even though it's highlighted our hint text for date has already been sorted so no need to worry about that, it displays dynamically based on which month it is so it doesn't need to match what the spreadsheet has for it)